src/webext/_locales/de/messages.json

```diff
@@ -0,0 +1,29 @@
+{
+  "extensionDescription": {
+    "message": "Dieses Add-on fügt einen Knopf zum öffnen mehrerer URLs aus der Zwischenablage hinzu."
+  },
+  "pasteAndGoButtonLabel": {
+    "message": "Einfügen & Los"
+  },
+  "switchToNewTabsLabel": {
+    "message": "Neue Tabs im Vordergrund öffnen"
+  },
+  "discardNewTabsLabel": {
+    "message": "Neue Tabs nicht bis zur Auswahl laden"
+  },
+  "heuristicModeLabel": {
+    "message": "URLs versuchen zu analysieren"
```

Comment:
i'm not sure about this translation, because i also don't really have a clue what this setting (*"Heuristic parsing of URL"*) does.
maybe @wantora can elaborate a little bit on this?

Reply:
"heuristicMode" is a setting to fix broken URLs.
Thanks for the translation!

Reply:
well... in this case i would use `"Versuche URLs zu reparieren"`, which more or less literally translates to `"Try to fix URLs"`. and because hardly any casual user really gets what *"Heuristic parsing"* means, i would also consider changing the original/english wording.

if this (translation) is not accurate and you are not ok with it, please @wantora elaborate a little bit more what this option (technically) does.

Reply:
@wantora this one translation is still crappy IMHO... are you ok with the mentioned ^ translation?

Reply:
That's fine. Thank you!